src/components/SideNavMobile.tsx

```diff
@@ -62,7 +62,7 @@ const LinkContainer: React.FC<{ children: ReactNode }> = ({ children }) => {
   )
 }
 
-const SideNavLink: React.FC<ILinkProps> = ({ children, ...props }) => {
+const SideNavLink = ({ children, ...props }) => {
```

Comment:
not sure about this one but I think this should still work
```suggestion
const SideNavLink = ({ children, ...props }: ILinkProps) => {
```